src/app/api/credmanager/route.ts

```diff
@@ -10,27 +11,23 @@ export interface UserCredDto {
 }
 
 export async function GET(req: NextRequest) {
-  // try {
   const teamId = req.nextUrl.searchParams.get("team_id");
   const session = await getServerSession(options);
   if (teamId && session?.userId != null) {
-    const CRED_MANAGER_ROUTE = process.env.CRED_MANAGER_ROUTE;
-    const calculatedUserCredDtos = await fetch(
-      `${CRED_MANAGER_ROUTE}/cred/team/${teamId}/${session?.accessToken}/${session?.user?.email}`,
-      {
-        method: "GET",
-      },
-    );
     createEvent(session?.userId, EventName.REQUEST_CALCULATION, {
       teamId: teamId,
     });
+
+    const contributionRequest = await createContributionRequest(
+      teamId,
+      session.accessToken!,
+      session!.user!.email!,
```

Comment:
let's check that the session.user exist with an if condition, using "!" on the backend could cause several issues